src/Interpreters/SystemLog.cpp

```diff
@@ -232,7 +232,10 @@ SystemLogs::SystemLogs(ContextPtr global_context, const Poco::Util::AbstractConf
     if (zookeeper_log)
         logs.emplace_back(zookeeper_log.get());
     if (session_log)
+    {
         logs.emplace_back(session_log.get());
+        global_context->addWarningMessage("Table system.session_log exists. It's unreliable and may contain garbage. Do not use it for any kind of security monitoring.");
```

Comment:
It may exist on startup as well as other `system.*_log` tables. The problem is not that `system.session_log` exists, but that someone may try to use it for security monitoring. This log is unreliable and should not be used for any security purposes, users should not trust content of this table. That's what this warning about.